core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -702,6 +634,93 @@ private[spark] class MemoryStore(
   }
 }
 
+private trait ValuesBuilder[T] {
+  def preciseSize: Long
```

Comment:
Hey guys, why not name the trait as `MemoryEntryBuilder`?  As I see from the code, it is used to build the `MemoryEntry`.

Reply:
good idea